gatsby/content/blog/2020/12/2020-12-07-gitter-now-speaks-matrix.mdx

```diff
@@ -12,8 +12,8 @@ Hi all,
 It’s been just over 2 months since we revealed that [Gitter was going to join Matrix](https://matrix.org/blog/2020/09/30/welcoming-gitter-to-matrix) - and
 we are incredibly proud to announce that Gitter has officially turned on true native Matrix connectivity:
 **all public Gitter rooms are now available natively via Matrix, and all Gitter users now natively exist on Matrix.**
-So, if you wanted to join the official Scala language support room at [https://gitter.im/scala/scala](https://gitter.im/scala/scala)
-from Matrix, just head over to [#scala_scala:gitter.im](https://matrix.to/#/#scala_scala:gitter.im) and \*boom\*, you’re in!
+So, if you wanted to join the official Scala language support room at [https://gitter.im/scala/center](https://gitter.im/scala/center)
```

Comment:
I assume you're after a Gitter room that shows the community name different from the room name to make the format more obvious?

Reply:
Exactly, this was the idea of this PR. I just looked for another rather generic Gitter room under Scala.